include/Util/TransparentImage.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef UTIL_TRANSPARENT_IMAGE_HPP
+#define UTIL_TRANSPARENT_IMAGE_HPP
+
+#include <string>
+#include <SDL.h>
+#include <SDL_image.h>
+#include <iostream>
+
+#include "Core/TransparentImageBase64.hpp"
+#include "Base64.hpp"
+
+static constexpr auto TRANSPARENT_IMAGE_BASE64_DECODE_LENGTH = DecodeBase64Length(TRANSPARENT_IMAGE);
+static constexpr auto TRANSPARENT_IMAGE_BASE64_DECODE = DecodeBase64<TRANSPARENT_IMAGE_BASE64_DECODE_LENGTH>(TRANSPARENT_IMAGE);
+
+SDL_Surface* GetTransparentImageSDLSurface(){
+    SDL_RWops* rwop = SDL_RWFromConstMem(TRANSPARENT_IMAGE_BASE64_DECODE.data(), TRANSPARENT_IMAGE_BASE64_DECODE.size());
+    SDL_Surface* aSurface = IMG_LoadTyped_RW(rwop, 1, "PNG");
+
+    if (aSurface == nullptr) {
+        LOG_ERROR("base64ToSurface");
```

Comment:
you should `#include <Util/Logger.hpp>` but I'm not sure why this compiles

Reply:
because this `hpp` only included in `Image.cpp` after somewhere `#include <Util/Logger.hpp` (I guess)
https://github.com/ntut-open-source-club/practical-tools-for-simple-design/blob/f5d08730459aff676e9fc7b1fe08bc955d18a524/src/Util/Image.cpp#L1-L9

Reply:
Fix on 583aeaf